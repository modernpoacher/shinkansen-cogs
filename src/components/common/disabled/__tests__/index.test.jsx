import React from 'react'
import renderer from 'react-test-renderer'

import Component from 'shinkansen-cogs/components/common/disabled'

jest.useFakeTimers()

describe('shinkansen-cogs/components/common/disabled', () => {
  describe('<Component />', () => {
    it('renders', () => {
      return expect(renderer.create(<Component />).toJSON())
        .toMatchSnapshot()
    })
  })
})
