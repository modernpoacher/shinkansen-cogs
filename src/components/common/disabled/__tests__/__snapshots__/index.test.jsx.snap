// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shinkansen-cogs/components/common/disabled <Component /> renders 1`] = `
<span
  className="is-disabled"
>
  *
</span>
`;
